testing-guidance/workflow-tests/track-view/track-view-tests.md

```diff
@@ -0,0 +1,88 @@
+## **Track View Workflow Tests**
+
+Testing in this area should focus on the functionality of the Track View. This is a general tool for the creating, editing and exporting of cutscenes.
+
+## General Docs
+
+* https://www.o3de.org/docs/user-guide/visualization/cinematics/track-view/
+##  Common Issues to Watch For
+
+Test guidance will sometimes note specific issues to watch for. The common issues below should be watched for through all testing, even if unrelated to the current workflow being tested.
+
+* The Camera parameter does not take effect.
+* Track View keyframe attributes cannot be modified.
+* Nodes cannot be added to the Sequence.
+* Unable to switch between multiple cameras.
+* Editor or Track View crashes.
+* Track View does not display the recorded Sequence.
+
+# Workflows
```

Comment:
nit: Only use single '#' for document title -> this should be updated to '##' most likely (and other headings should maybe change to '###')